libs/ledger-live-common/src/platform/logic.ts

```diff
@@ -144,8 +144,6 @@ export type CompleteExchangeRequest = {
   signature: string;
   feesStrategy: string;
   exchangeType: number;
-  swapId?: string;
-  rate?: number;
```

Comment:
Shouldn't we keep that ?
We keep it for wallet-api and you use it for the request in the Body component